src/Acuminator/Acuminator.Utilities/Common/StringExtensions.cs

```diff
@@ -54,6 +54,21 @@ public static bool IsEmpty(this string source)
 				: char.ToUpperInvariant(s[0]).ToString();
 		}
 
+		// <summary>
+		/// A string extension method that converts the string to a camel case (first letter to lower case).
+		/// </summary>
+		/// <param name="s">The string to act on.</param>
+		/// <returns/>
+		public static string? ToCamelCase(this string? s)
```

Comment:
I would argue that it is more like "lowercase the first letter" because `ToCamelCase("snake_case")` doesn't return `"snakeCase"`, but it's not critical, considering the usages.

Reply:
Didn't know that camel case forbids the underscore, renamed the method